pkg/service/search.go

```diff
@@ -0,0 +1,67 @@
+package service
+
+import (
+	"fmt"
+	"github.com/coralproject/pillar/pkg/model"
+	"github.com/coralproject/pillar/pkg/web"
+	"gopkg.in/mgo.v2/bson"
+	"net/http"
+	"time"
+)
+
+// GetSearches returns the list of all Search items in the system
+func GetSearches(context *web.AppContext) ([]model.Search, *web.AppError) {
+
+	all := make([]model.Search, 0)
+	if err := context.DB.Searches.Find(nil).All(&all); err != nil {
+		message := fmt.Sprintf("Error fetching searches")
+		return nil, &web.AppError{err, message, http.StatusInternalServerError}
+	}
+
+	return all, nil
+}
+
+// CreateUpdateSearch upserts a Search
+func CreateUpdateSearch(context *web.AppContext) (*model.Search, *web.AppError) {
+	var input model.Search
+	context.Unmarshall(&input)
+
+	fmt.Printf("Search: %v", input)
```

Comment:
We should revisit the logging discussion again.  @gabelula found https://github.com/Sirupsen/logrus which will allow us to create multiple logging channels.  For now, simple output goes to loggly, so this is ok.